qml/PingSlider.qml

```diff
@@ -24,6 +25,73 @@ RowLayout {
         Layout.fillWidth: true
         stepSize: 1
         wheelEnabled: true
+
+        background: Rectangle {
+            x: sliderControl.leftPadding
+            y: sliderControl.topPadding + sliderControl.availableHeight/2 - height/2
+            implicitWidth: 200
+            implicitHeight: 1
+            width: sliderControl.availableWidth
+            height: implicitHeight
+            color: Material.accent
+
+            Repeater {
+                id: internalTicks
+                model : sliderControl.to - sliderControl.from
+
+                delegate: Rectangle {
+                    id: tick
+                    width: 1
+                    visible: {
```

Comment:
can you calculate the correct amount in `model` ? If the ticks does not need to exist, the don't need to.